app/models/vm/operations.rb

```diff
@@ -14,7 +14,14 @@ module Vm::Operations
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddresses.first, :port => 9090).to_s
+    URI::HTTP.build(:host => get_ipv4_address, :port => 9090).to_s
```

Comment:
A miss from local rubocop cleanup, doh!  I figured for ipv6 only connectivity the method falls back to the current functionality of using the first ip address.